packages/typespec-ts/src/transform/transformResponses.ts

```diff
@@ -185,9 +185,9 @@ function transformBody(
       descriptions.add("Value may contain any sequence of octets");
       continue;
     }
-    const bodySchema = getSchemaForType(dpgContext, body!.type, [
-      SchemaContext.Output
-    ]) as Schema;
+    const bodySchema = getSchemaForType(dpgContext, body!.type, {
```

Comment:
i don't think this is clearly defined and currently i am collecting the real cases and have an issue for that: https://github.com/Azure/autorest.typescript/issues/2276.

currently we would trust the @encode for bytes in response. if no defined it would go to default.